drivers/hyperv/powershell.go

```diff
@@ -69,6 +69,36 @@ func hypervAvailable() error {
 }
 
 func isAdministrator() (bool, error) {
+	var hypervAdmin bool
+	var windowsAdmin bool
+	var windowsErr error
+
+	hypervAdmin = isHypervAdministrator()
+
+	if hypervAdmin == true {
+		return true, nil
+	}
+
+	windowsAdmin, windowsErr = isWindowsAdministrator()
+
+	if windowsErr != nil {
+		return false, windowsErr
+	}
+
+	return windowsAdmin, nil
+}
+
+func isHypervAdministrator() bool {
+	stdout, err := cmdOut(`@([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole("Hyper-V Administrators")`)
+	if err != nil {
```

Comment:
Why not also `log.Debug(err)`?  There are quite a few reasons this might return an error which don't indicate permissions status.